tests/epyccel/modules/arrays.py

```diff
@@ -1828,6 +1828,14 @@ def iterate_slice(i : int):
         res += ai
     return res
 
+#==============================================================================
+# TESTS: VIEW
+#==============================================================================
+
+@types('int[:]', 'int')
+def array_1d_view_assign(x, a):
+    x[:] = a
```

Comment:
Can you please add a function that works on 2D views?

Reply:
https://github.com/pyccel/pyccel/pull/1299/commits/8a6e2b3cd1ae93ebef9c0a56fcc55f1efcee9c5c